lib/widgets/balance.dart

```diff
@@ -37,6 +36,8 @@ class BalanceState extends State<Balance> {
         builder: (BuildContext context, DashboardState state) {
       Account currentAccount =
           Locator.instance.get<ApiDatabase>().walletStorage.currentAccount;
+      Wallet currentWallet =
```

Comment:
Is it worth it to use a getter for wallet storage?
```WalletStorage get walletStorage => Locator.instance.get<ApiDatabase>().walletStorage;```

```suggestion
            Account currentAccount =
          walletStorage.currentAccount;
      Wallet currentWallet =
          walletStorage.currentWallet;
```